packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/CustomUserControlViewManagerCS.cs

```diff
@@ -20,7 +20,7 @@ public override FrameworkElement CreateView()
             {
                 if (obj is CustomUserControlCS c)
                 {
-                    LabelChanged(c, c.Label);
+                    LabelChanged?.Invoke(c, c.Label);
```

Comment:
Did you see a runtime error or it is a static analyzer issue?

Reply:
Just code cleanliness and habit. 
In previous project we started migrating to use C# [nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references). 
We assign it by default to null, so technically it can be null as `ViewManagerEvent` is a class..

---
In reply to: [420390563](https://github.com/microsoft/react-native-windows/pull/4793#discussion_r420390563) [](ancestors = 420390563)